dev-requirements.txt

```diff
@@ -1,8 +1,8 @@
 # install latest changes in dbt-core + dbt-postgres
 # TODO: how to switch from HEAD to x.y.latest branches after minor releases?
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-core&subdirectory=core
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-tests-adapter&subdirectory=tests/adapter
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-postgres&subdirectory=plugins/postgres
+git+https://github.com/dbt-labs/dbt-core.git@mcknight/convert-column_type#egg=dbt-core&subdirectory=core
+git+https://github.com/dbt-labs/dbt-core.git@mcknight/convert-column_type#egg=dbt-tests-adapter&subdirectory=tests/adapter
+git+https://github.com/dbt-labs/dbt-core.git@mcknight/convert-column_type#egg=dbt-postgres&subdirectory=plugins/postgres
 
```

Comment:
This very well might be my ignorance. Is this done just for development and then we change it back before merging into `main`? I noticed it on all three related PRs, so I assume it's intentional.

Reply:
yes, since the tests are conversions for a non merged test we do this so we can pip install that branch to test locally and show it is passing ci/cd for review, revert back to original pointers before merger.